### DISCUSSION TITLE
Prefetch and cache only to disk

### DISCUSSION THREAD

**Discussion by rdsarna** [onevcat/Kingfisher#2238] [Q&A] [answered]
Hi, is there a way to use the `ImagePrefetcher` and store the images only in the disk storage and not the memory? I went through the `KingfisherOptionsInfoItem` but could not find a case that would support this specifically other than creating a custom `ImageCache` and setting the `.targetCache`. Thanks in advance!

**Comment by onevcat** [comment, marked as answer]
Hi, @rdsarna 

Maybe you can try `.memoryCacheExpiration(.expired)`.

**Reply by rdsarna** [reply]
Thanks, this worked!
